src/Sql/Sql/Database/Cmdlet/SetAzureSqlDatabase.cs

```diff
@@ -257,6 +257,13 @@ public class SetAzureSqlDatabase : AzureSqlDatabaseCmdletBase<IEnumerable<AzureS
             HelpMessage = "The Maintenance configuration id for the SQL Database.")]
         public string MaintenanceConfigurationId { get; set; }
 
+        /// <summary>
+        /// Gets or sets the ledger option to assign to the Azure SQL Database
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "The enable ledger option for the Azure Sql Database")]
+        public SwitchParameter EnableLedger { get; set; }
```

Comment:
Since we don't allow alterting this, should this be part of the Set cmdlet?

Reply:
yes this is true, let me remove it